lib/scenic/adapters/postgres.rb

```diff
@@ -144,17 +147,19 @@ def create_materialized_view(name, sql_definition)
       #
       # @param name The name of the view to update
       # @param sql_definition The SQL schema for the updated view.
+      # @param no_data [Boolean] Default: false. Set to true to create materialized view fast w/o
+      # feeding it with data. You will then need to `REFRESH MATERIALIZED VIEW`
       #
       # @raise [MaterializedViewsNotSupportedError] if the version of Postgres
       #   in use does not support materialized views.
       #
       # @return [void]
-      def update_materialized_view(name, sql_definition)
+      def update_materialized_view(name, sql_definition, no_data = false)
```

Comment:
Let's use a keyword argument here as well. I think it will improve readability at call sites.